connector_helpscout/models/helpscout_customer/importer.py

```diff
@@ -27,6 +27,22 @@ def email(self, record):
             return
 
     @mapping
+    @only_create
+    def odoo_id(self, record):
+        # Searches res.partner records for matching email address,
+        # excluding internal users
+        try:
+            customer = self.env['res.partner'].search([
+                ('email', '=', record.emails[0].value),
```

Comment:
Tag team off of the pre-existing email method here, just in case we decide to change something. `self.email(record)['email']` instead of `record.email...`